nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -1467,4 +1498,11 @@ public boolean isProcessRunning() {
             return Boolean.TRUE.equals(processRunning);
         }
     }
+
+    private static class NiFiNotRunningException extends Exception {
+        @Override
+        public Throwable fillInStackTrace() {
```

Comment:
Would you make this synchronized to match parent class implementation?

Reply:
Yes, thanks!